### PR TITLE
more bottles

### PR DESCRIPTION
**Pull request by UnixJunkie**
Please use the [proposal template](/proposal_template.md) for your pull request.

### CODE REVIEW

File: proposal_template.md [hunk 1]
```diff
@@ -1,24 +1,18 @@
...
-Describe a solution to the problem. Provide examples and describe how they
-work. Show how your solution is better than current workarounds: is it cleaner,
-safer, or more efficient?
+Create as many bottles as necessary and as possible.
 
 ## Detailed design
-Describe the design of the solution in detail. The detail in this section
-should be sufficient for someone who is *not* one of the authors of this
-proposal to be able to reasonably implement the feature.
+I dont' know.
```
Comment (vitorgalvao) [inline proposal_template.md]: Typo (should be “don’t”).

File: proposal_template.md [hunk 2]
```diff
@@ -1,24 +1,18 @@
 # $FEATURE_NAME
```
Comment (vitorgalvao) [inline proposal_template.md]: You forgot the title.

**Comment by vitorgalvao** [comment]
You’re not supposed to edit the `proposal_template.md` itself, but to make a copy of it for your proposal. As it stands, you would be removing the template for other people.

As for the proposal itself, you may want to revise it. Currently it is the same as saying “implement more features” without being specific about what those are or how they’d work. Adding bottles is on-going work. It’s done as the rhythm time and resources allow — see the [Bottles documentation](https://github.com/Homebrew/brew/blob/master/docs/Bottles.md) for details on how they’re created.

Simply asking for more bottles does not produce an effect, and is not a tangible improvement to the software itself (i.e. the kind of proposal adequate for this repository). A better approach would be to ask first “why aren’t there any more bottles?” and possibly even offering your help if the response is something you are able to help on.

**Comment by MikeMcQuaid** [comment]
> Create as many bottles as necessary and as possible.

We do already.

**Pull request closed without merging**